.github/workflows/release-vm-console-proxy.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
           OUTPUT_FILE=./data/vm-console-proxy-bundle/vm-console-proxy.yaml
           mkdir -p ./data/vm-console-proxy-bundle
           curl -L https://github.com/kubevirt/vm-console-proxy/releases/download/${RELEASE_VERSION}/vm-console-proxy.yaml > ${OUTPUT_FILE}
+          sed -i "s/defaultVmConsoleProxyImageTag = .*$/defaultVmConsoleProxyImageTag = \"${RELEASE_VERSION}\"/" ./internal/operands/vm-console-proxy/defaults.go 
```

Comment:
IMHO, I don't think we should do that bump here.

1. This workflow is used to update vm-console-proxy bundle only
2. We try to change code during that workflow, what if `defaults.go` will be removed in the future?

Why not just use `latest` tag? if anyways this intended to be updated on every release to latest one, just use `latest` then: `quay.io/kubevirt/vm-console-proxy:latest`.

Reply:
> 1. This workflow is used to update vm-console-proxy bundle only

This workflow is used to update SSP operator code, so that it uses the latest release of `vm-cosole-proxy`. It can update the image tag and bundle.

> 2. We try to change code during that workflow, what if `defaults.go` will be removed in the future?

I will add a comment to `defaults.go` explaining how the file is used.

> Why not just use `latest` tag? if anyways this intended to be updated on every release to latest one, just use `latest` then: `quay.io/kubevirt/vm-console-proxy:latest`.

There is no `latest` tag. We would need to add automation to create and update it.
https://quay.io/repository/kubevirt/vm-console-proxy?tab=tags&tag=latest

Using the latest tag would probably not save us work, because we would need to fix the version when creating a release branch.